apps/passport-server/src/database/queries/telegram/fetchTelegramEvent.ts

```diff
@@ -60,3 +60,25 @@ export async function fetchLinkedPretixAndTelegramEvents(
 
   return result.rows;
 }
+
+export interface ChatIDWithEvents {
+  telegramChatID: string;
+  ticketEventIds: string[];
+}
+
+export async function fetchEventsPerChat(
```

Comment:
The new query as discussed with @rrrliu. We identify groups of events based on the telegram chat ID they are linked to. This requires careful management of linking events with telegram chats.